solutions/swb-app/src/staticPermissionsConfig.ts

```diff
@@ -14,37 +14,42 @@ const adminPermissions: Permission[] = [
   {
     effect: 'ALLOW',
     action: 'CREATE',
-    subject: 'Environment'
```

Comment:
did you add new ones and git took it as double update instead?

Reply:
I alphabetized all of the permissions by subject, thats why these look funny.